packages/main/src/plugin/color-registry.ts

```diff
@@ -331,6 +331,15 @@ export class ColorRegistry {
       dark: colorPalette.white,
       light: colorPalette.purple[900],
     });
+
+    this.registerColor('titlebar-windows-hover-exit-bg', {
+      dark: '#be4425',
+      light: '#be4425',
+    });
+    this.registerColor('titlebar-windows-hover-bg', {
+      dark: '#2d2d2d',
+      light: '#2d2d2d',
```

Comment:
is it expected that colors for light and dark mode are the same ?

Reply:
We could stick with - hover:charcoal-300 for light mode (Linux has those colors)

Reply:
Light theme hover on non-exit buttons: gray-400:

https://github.com/user-attachments/assets/9fc5a230-24b5-4881-8cf4-f8cc6c0074b1